app/Views/admin/products/Products.php

```diff
@@ -98,6 +98,7 @@
                     <!-- card-body -->
                 </div>
                 <!-- card -->
+                <?= view('templates/admin/deleteModalElement', ['idModal' => 'deleteProductsModal', 'message' => 'Eliminar el producto resultará en la eliminación permanente del elemento. Esta acción no se puede deshacer y conlleva la eliminación de los archivos relacionados y los tags asociadas al producto.', 'action' => '/admin/productos/borrar', 'inputName' => 'productId']); ?>
```

Comment:
I think the message looks better like this:

```suggestion
                <?= view('templates/admin/deleteModalElement', ['idModal' => 'deleteProductsModal', 'message' => 'Eliminar el producto resultará en la eliminación permanente del elemento. Esta acción no se puede deshacer e implica la eliminación de los archivos y los tags asociados al producto.', 'action' => '/admin/productos/borrar', 'inputName' => 'productId']); ?>
```